eq-author-api/app.js

```diff
@@ -1,14 +1,16 @@
 const express = require("express");
 const { graphqlExpress } = require("graphql-server-express");
-const repositories = require("./repositories");
 const cors = require("cors");
 const bodyParser = require("body-parser");
-const schema = require("./schema");
 const pinoMiddleware = require("express-pino-logger");
+
 const { PORT } = require("./config/settings");
 const status = require("./middleware/status");
 const { getLaunchUrl } = require("./middleware/launch");
 const createAuthMiddleware = require("./middleware/auth");
+const repositories = require("./repositories");
+const modifiers = require("./modifiers");
```

Comment:
Granted I haven't looked through them all yet, but from the first couple of examples that I've seen these modifiers don't appear to be "modifying" anything. Rather they appear to be orchestrating repositories, performing validation and calling other modifiers. I think `services` is a more conventional name for this type of entity, to me the name modifiers is a bit misleading and confusing. But I might be missing something that I haven't got to yet.

Reply:
We did have a bit of a discussion about the name and it could probably be improved. However, they are always used for performing changes; creation, update and updating based on other events. They are never used for reading which is done directly against the repositories (at the moment). Where as a service I would view that as something that is completely isolated and completely handles that entity.

Also, a side note, service is an overloaded term that has multiple meaning including; separate network app or an OS managed daemon.

Reply:
Fair enough. I can live with it https://martinfowler.com/bliki/TwoHardThings.html